sample-deployments/composer-orchestrated-process/DEPLOYMENT.md

```diff
@@ -1,68 +1,4 @@
-# Deployment Guide
+# [Deployment Guide](../../README.md#deployment-guide)
```

Comment:
Is there a `#deployment-guide` in the `../../README.md` file? I can't see it

Reply:
Yes, https://github.com/GoogleCloudPlatform/document-processing-and-understanding/blob/delete-doc/README.md#deployment-guide